### DISCUSSION TITLE
Markdown doesn't render equations inside bullet list items

### DISCUSSION THREAD

**Discussion by bachmeil** [community/community#16958] [Other Feature Feedback, Questions, & Ideas] [answered]
There's now support for mathjax inside markdown, but it doesn't render inside bullet list items. Example:

```
List:

- $y_{t}$
- Doesn't work

$y_{t}$ works
```

**Comment by afourmy** [comment]
It doesn't render inside headings either (example: `## $k$`)

**Comment by nschloe** [comment, marked as answer]
See [my blog post](https://nschloe.github.io/2022/05/20/math-on-github.html) on why this happens.

tldr; Markdown and Math renderer are competing, leading to many such bugs.
